src/activeTextEditorState.ts

```diff
@@ -0,0 +1,12 @@
+import * as vscode from "vscode";
+import { prefetch } from "./binary/requests/prefetch";
+
+// Notify the binary on changes to the active text editor.
+export const activeTextEditorState = vscode.window.onDidChangeActiveTextEditor(
```

Comment:
if i remember correctly this will be fired on any scheme(git diff etc),  can you check this? an if so filter only the file scheme :
```
document.uri.scheme === "file"
```

Reply:
validated and it gets the file name.